nyt.py

```diff
@@ -10,9 +10,10 @@
 from sqlalchemy.orm.exc import (
```

Comment:
Yeeees.... I think I wanted to use the NYT integration in metadata, but it's like Novelist in that the licensing prohibits sharing of the best-seller list data. And the title-author type metadata that comes from this source is of very low quality, so we wouldn't want to use it internally.